src/registrar/templates/application_status.html

```diff
@@ -1,114 +1,99 @@
 {% extends 'base.html' %}
 
-{% block title %}Domain request status- {{ domainapplication.requested_domain.name }}{% endblock %}
+{% block title %}Domain request status- {{ domainapplication.requested_domain.name }} | {% endblock %}
```

Comment:
**comment (non-blocking)**: For consistency, should we use the vertical bar for separating "Domain request status" from the domain name as we have on other pages?

Reply:
Great point, updated.